packages/cookie-consent-react/src/CookieConsentModal.tsx

```diff
@@ -0,0 +1,152 @@
+import React from "react";
+// @ts-ignore: wait for nrk to supply types
+import CoreDialog from "@nrk/core-dialog/jsx";
+import { CheckListItem, UnorderedList } from "@fremtind/jkl-list-react";
+import { Checkbox } from "@fremtind/jkl-checkbox-react";
+import { PrimaryButton, TertiaryButton } from "@fremtind/jkl-button-react";
+import { useForm } from "react-hook-form";
+import { Consent, ConsentComponentBaseProps } from "./types";
+import { useCookieConsentState } from "./CookieConsentContext";
+import { convertConsentValueToFormValue, convertBooleanToConsentValue } from "./cookieConsentUtils";
+
+interface FormValues {
+    functional?: boolean;
+    marketing?: boolean;
+    statistics?: boolean;
+}
+
+export const CookieConsentModal = ({ onAccept }: ConsentComponentBaseProps) => {
```

Comment:
Dette ser ut som det eneste stedet du bruker `ConsentComponentBaseProps`.. Går ut fra at den skal speile `onAccept` fra `CoockieConsent`? I så fall vil jeg kanskje foreslå å definere den her, og hente den ut i `CookieConsent` med `React.ComponentProps`. Samme type bør i hvert fall brukes begge steder, så man sørger for at de er i synk 😊

Reply:
`ConsentBaseProps` var i utgangspunktet et større interface med mer funksjonalitet. Tanken var at alle som implementerer en variant av cookie consent ala modalen kunne enkelt dra inn dette interfacet så det var en standard måte å sette opp UI-komponentene.

Enig i at en felles type for et accept-callback gir mening 😄